static/style/search.css

```diff
@@ -306,10 +306,16 @@ form.compact {
       outline-offset: 2px;
     }
 
-    &.is-active {
+    &[aria-pressed='true'] {
       color: var(--foreground-1);
       font-weight: 500;
       text-decoration: none;
     }
+
+    &[disabled] {
+      cursor: default;
+      pointer-events: none;
```

Comment:
TIL about [inert](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Global_attributes/inert), which is a bit more comprehensive than just `pointer-events`. This is fine though, the combination of `:disabled` and removing `pointer-events` does about the same. Just wanted to share that :)